contracts/epochs/FlowEpoch.cdc

```diff
@@ -338,6 +338,60 @@ pub contract FlowEpoch {
             FlowEpoch.account.load<Bool>(from: /storage/flowAutomaticRewardsEnabled)
             FlowEpoch.account.save(enabled, to: /storage/flowAutomaticRewardsEnabled)
         }
+
+        /// Protocol can use this to reboot the epoch with a new genesis
+        /// in case the epoch setup phase did not complete properly
+        /// before the end of an epoch
```

Comment:
```suggestion
        /// Ends the currently active epoch and starts a new one with the provided configuration.
        /// The new epoch, after resetEpoch completes, has `counter = currentEpochCounter + 1`.
        /// This function is used during sporks - since the consensus view is reset, and the protocol is
        /// bootstrapped with a new initial state snapshot, this initializes FlowEpoch with the first epoch
        /// of the new spork, as defined in that snapshot.
```